react-native-fbsdkcore/js/FBSDKGraphRequest.ios.js

```diff
@@ -17,69 +17,71 @@
 * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 *
-* @providesModule FBSDKGraphRequest
 * @flow
 */
 
 'use strict';
 
-/*
+var FBSDKGraphRequestManager = require('./FBSDKGraphRequestManager.ios.js');
+
+type FBSDKGraphRequestCallback = (error: ?Object, result: ?Object) => void;
+type FBSDKGraphRequestParameters = {[key: string]: Object};
```

Comment:
Actually, to be more correct, `{string?: string, uri?: string}`. Note the placement of the `?`. Also `string`, not `Object` ;). Although the most correct form would be a union: `{string: string} | {uri: string}`, but last I tried, Flow doesn't like objects in union types unless there's a property that identifies the disjoint union type with an elementary value (e.g. a string or a number). See http://flowtype.org/blog/2015/07/03/Disjoint-Unions.html

Anyway, that API is going to be in flux somewhat, so I didn't want to dig too deep into stuff that's possibly going to change soon.